### PR TITLE
[chess] So long, and thanks for all the fish!

### PR DESCRIPTION
**Pull request by cognigami**
Quantum Chess' move to //unitary is complete.

### CODE REVIEW

**Comment by cognigami** [comment]
By way of heads-up: I didn't have rights to remove the two references to quantum chess in .github/workflows:
 - black.yml
 -  pythonpackage.yml

**Comment by dstrain115** [comment]
Those two files are in the repository, so you should be able to edit them and submit them as part of the PR.
https://github.com/quantumlib/ReCirq/tree/master/.github/workflows

What permission errors are you seeing?

**Comment by mpharrigan** [comment]
Woot! Maybe we should leave the `recirq/quantum_chess` directory with a README pointing to the new location. We can also have an `__init__.py` that raises an `ImportError` telling you to use the new package

**Comment by cognigami** [comment]
Thanks @dstrain115, my token didn't have workflow permissions. I've pushed the commit now. Just waiting on advice on merging then patching PR #298 in to this PR.

File: .github/workflows/pythonpackage.yml [hunk 1]
```diff
@@ -50,12 +50,6 @@ jobs:
           # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
           flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
 
-      - name: Test with pytest
-        run: |
-          pip install pytest
-          # RECIRQ_CHESS_TEST_SEED: make quantum_chess tests determnistic
-          # RECIRQ_IMPORT_FAILSAFE: skip tests on unsupported Cirq configurations
-          RECIRQ_CHESS_TEST_SEED=789 RECIRQ_IMPORT_FAILSAFE=y pytest -v
```
Comment (mpharrigan) [inline .github/workflows/pythonpackage.yml:58]: this disables pytest for the whole package. remove the chess_test_seed but we still need the call to `pytest`

**Review by mpharrigan — CHANGES_REQUESTED** [review]

**Comment by mpharrigan** [comment]
@cognigami what do you think about my suggestion:

> Maybe we should leave the recirq/quantum_chess directory with a README pointing to the new location. We can also have an __init__.py that raises an ImportError telling you to use the new package

File: quantum_chess/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Quantum Chess no longer exists in this repo. See README.md for mre info.
+
+raise ImportError
```
Comment (mpharrigan) [inline quantum_chess/__init__.py:3]: you need to `raise ImportError()` so it actually constructs the exception object.

You can also add a helpful message: `raise ImportError("quantum_chess moved to github.com/quantumlib/unitary/")`

File: .github/workflows/pythonpackage.yml (hunk 1 above)
Comment (mpharrigan) [reply .github/workflows/pythonpackage.yml:58]: @cognigami this is an important issue to fix before merge ^

Comment (mpharrigan) [reply .github/workflows/pythonpackage.yml:58]: still missing call to `pytest`

File: quantum_chess/__init__.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Quantum Chess no longer exists in this repo. See README.md for mre info.
+
+raise ImportError("quantum_chess moved to https://github.com/quantumlib/unitary/tree/main/unitary/quantum_chess")
```
Comment (mpharrigan) [inline quantum_chess/__init__.py:3]: this should be under `recirq/quantum_chess` where the previous `__init__.py` file was

File: quantum_chess/README.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+Quantum Chess has graduated to its own repository. You can follow the work on
```
Comment (mpharrigan) [inline quantum_chess/README.md:1]: this should move to `recirq/quantum_chess` where the code used to be

File: .github/workflows/pythonpackage.yml [hunk 5]
```diff
@@ -53,9 +53,7 @@ jobs:
       - name: Test with pytest
         run: |
           pip install pytest
-          # RECIRQ_CHESS_TEST_SEED: make quantum_chess tests determnistic
-          # RECIRQ_IMPORT_FAILSAFE: skip tests on unsupported Cirq configurations
-          RECIRQ_CHESS_TEST_SEED=789 RECIRQ_IMPORT_FAILSAFE=y pytest -v
```
Comment (mpharrigan) [inline .github/workflows/pythonpackage.yml:58]: this is a call to `pytest` (don't be confused by the preceding environment variable declarations). We cannot have this removed. It tests the whole repository.

File: .github/workflows/pythonpackage.yml [hunk 6]
```diff
@@ -53,9 +53,7 @@ jobs:
...
+
```
Comment (mpharrigan) [inline .github/workflows/pythonpackage.yml:56]:
```suggestion
          # RECIRQ_IMPORT_FAILSAFE: skip tests on unsupported Cirq configurations
          RECIRQ_IMPORT_FAILSAFE=y pytest -v
```

**Review by mpharrigan — CHANGES_REQUESTED** [review]
we need to still have the tests enabled for the rest of the repository

File: docs/_toc.yaml [hunk 7]
```diff
@@ -68,4 +68,4 @@ toc:
 
 - heading: "Rabi oscillations"
 - title: "Rabi Oscillation Experiment"
-  path: /cirq/experiments/benchmarks/rabi_oscillations
```
Comment (mpharrigan) [inline docs/_toc.yaml:71]: bad merge?

**Comment by mpharrigan** [comment]
This seems to revert the changes from #305 ?

**Comment by cognigami** [comment]
I think I made a bad merge. Let's see if I can say the right prayers to the git gods to undo this.

**Review by mpharrigan — APPROVED** [review]

**Pull request merged**